src/config/dynamic-theme-fixes.config

```diff
@@ -212,9 +219,19 @@ fivethirtyeight.com
 
 INVERT
 .logo
-.site-logo
-#searchform
-.header-espn-link
+
+CSS
+#searchform path,
+.header-espn-link path,
+.site-logo path {
+    fill: #fff;
```

Comment:
Please, pay attention to adding `!important` keyword. check that your fix looks well in both Dark and Light mode, special `${white}` template is used to apply filter values and inversion to specified color.